bitrise/status.go

```diff
@@ -0,0 +1,68 @@
+package bitrise
+
+import (
+	"fmt"
+	"github.com/bitrise-io/bitrise/models"
+)
+
+func HumanReadableStatus(status int) string {
```

Comment:
I did the custom string type and it was working great in the cli but I discovered a hidden dependency. The cli analytics plugin uses the same model. The analytics plugin reads the parameters from the stdin and serialises into the `StepRunResultsModel` field. Luckily the plugin converts this status field into the same human readable strings we were planning to use it. Now the only problem that the the plugin and the cli needs to be the same version (where this field is updated in both) otherwise the analytics plugin will fail to send the final events at the end of the bitrise run. I have found a plugin release confluence page and looked at the code and does not fully see yet how are the "breaking" changes are handled. 

So for now I will use a custom int field where I define the functions. And once we had a chat about I will create a follow up PR where I clean this up.